containers/support/BugModal.js

```diff
@@ -0,0 +1,185 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { reportBug } from 'proton-shared/lib/api/reports';
+import { getOS, getBrowser, getDevice } from 'proton-shared/lib/helpers/browser';
+import { c } from 'ttag';
+import {
+    Modal,
+    Href,
+    Alert,
+    Row,
+    Input,
+    Button,
+    useToggle,
+    Info,
+    TextArea,
+    Label,
+    ContentModal,
+    FooterModal,
+    EmailInput,
+    ResetButton,
+    PrimaryButton,
+    useApiWithoutResult,
+    useUser,
+    useAddresses,
+    useNotifications
+} from 'react-components';
+import AttachScreenshot from './AttachScreenshot';
+
+const BugModal = ({ show, onClose }) => {
+    const [{ Name = '' }] = useUser();
+    const { createNotification } = useNotifications();
+    const [addresses = []] = useAddresses();
+    const [{ Email = '' } = {}] = addresses;
+    const os = getOS();
+    const browser = getBrowser();
+    const device = getDevice();
+    const [model, update] = useState({
+        OS: os.name,
+        OSVersion: os.version || '',
+        Browser: browser.name,
+        BrowserVersion: browser.version,
+        Resolution: `${window.innerHeight} x ${window.innerWidth}`,
+        Client: 'Angular', // TODO define a new one with the API
```

Comment:
We should pass the config from the app as a prop to the modal, so we can extend it with specific keys

Reply:
A `Context` is probably more appropriate.